.nvmrc

```diff
@@ -1 +1 @@
-18.18
\ No newline at end of file
+18.17
```

Comment:
Can you add a comment to not update this value until https://github.com/microsoft/vscode/issues/194921 is resolved.

Reply:
I'm unsure whether comments can be added to the .nvmrc file
The following fail:
- https://dev.azure.com/monacotools/Monaco/_build/results?buildId=235342&view=logs&j=8e733ce0-0c7b-5d06-69ef-2aad0d062863&t=20dfcc3f-29fb-5b35-1948-57062942b9b4
- https://dev.azure.com/monacotools/Monaco/_build/results?buildId=235340&view=logs&j=8e733ce0-0c7b-5d06-69ef-2aad0d062863&t=20dfcc3f-29fb-5b35-1948-57062942b9b4

Reply:
Thanks for checking